Example/ViewController.swift

```diff
@@ -48,7 +54,237 @@ class ViewController: UIViewController {
             startButton.isEnabled = true
             mapView?.show(routes)
             mapView?.showWaypoints(on: currentRoute)
+            if let style = mapView?.style {
+                // show route Duration and Toll annotations
+                updateRouteAnnotations(routes, style: style)
+            }
+        }
+    }
+
+    fileprivate let dateComponentsFormatter = DateComponentsFormatter()
+
+    private func updateAnnotationSymbolImages(_ style: MGLStyle) {
+        let capInsetHeight = CGFloat(22)
+        let capInsetWidth = CGFloat(11)
+        let capInsets = UIEdgeInsets(top: capInsetHeight, left: capInsetWidth, bottom: capInsetHeight, right: capInsetWidth)
+        if let image = UIImage(named: "RouteInfoAnnotationLeftHanded") {
+            let regularRouteImage = image.tint(UIColor.white).resizableImage(withCapInsets: capInsets, resizingMode: .stretch)
+            style.setImage(regularRouteImage, forName: "RouteInfoAnnotationLeftHanded")
+
+            let selectedRouteImage = image.tint(#colorLiteral(red: 0.337254902, green: 0.6588235294, blue: 0.9843137255, alpha: 1)).resizableImage(withCapInsets: capInsets, resizingMode: .stretch)
+            style.setImage(selectedRouteImage, forName: "RouteInfoAnnotationLeftHanded-Selected")
+        }
+
+        if let image = UIImage(named: "RouteInfoAnnotationRightHanded") {
+            let regularRouteImage = image.tint(UIColor.white).resizableImage(withCapInsets: capInsets, resizingMode: .stretch)
+            style.setImage(regularRouteImage, forName: "RouteInfoAnnotationRightHanded")
+
+            let selectedRouteImage = image.tint(#colorLiteral(red: 0.337254902, green: 0.6588235294, blue: 0.9843137255, alpha: 1)).resizableImage(withCapInsets: capInsets, resizingMode: .stretch)
+            style.setImage(selectedRouteImage, forName: "RouteInfoAnnotationRightHanded-Selected")
+        }
+    }
+
+    private func updateRouteAnnotations(_ routes: [Route]?, style: MGLStyle) {
+        // remove any existing route annotation
+        removeRouteAnnotationsLayerFromStyle(style)
```

Comment:
When route is selected `ViewController.response` will trigger `clearMapView()`, which will in turn clear delay view. Since this feature is going to be part of `MapboxNavigation` it makes sense to move existing logic into `NavigationMapView` instead of improving implementation in Example app.

Reply:
It'd be also good to rebase current changes onto `main` as it now contains at least one fix to example app which can be useful here (previously after pressing on route clear button would disappear).

Reply:
Completed the rebase.